internal/controller/operator/deployment.go

```diff
@@ -1063,6 +1063,42 @@ func buildIdpVolumes(ldapCACert string, routerCertSecret string, auditSecretName
 				},
 			},
 		},
+		{
+			Name: "liberty-serverdir-vol",
+			VolumeSource: corev1.VolumeSource{
+				EmptyDir: &corev1.EmptyDirVolumeSource{},
+			},
+		},
+		{
+			Name: "liberty-outputdir-vol",
+			VolumeSource: corev1.VolumeSource{
+				EmptyDir: &corev1.EmptyDirVolumeSource{},
+			},
+		},
+		{
+			Name: "liberty-logs-vol",
+			VolumeSource: corev1.VolumeSource{
+				EmptyDir: &corev1.EmptyDirVolumeSource{},
+			},
+		},
+		{
+			Name: "liberty-tmp-vol",
```

Comment:
What are we storing in /tmp?

Reply:
When liberty starts up, it writes some data to /tmp directory, without this volume, liberty server doesn't come up